services-manager/src/main/resources/application-services.yaml

```diff
@@ -29,7 +29,7 @@ manager:
   default:
     # Library includes relative to manager.root above (e.g. ../palisade-examples => palisade-services/../palisade-examples)
     # While the example library happens to be the same version as services, it should be declared here as a static name
-    library: ../palisade-examples/example-library/target/
+    library: ../Palisade-examples/example-library/target/example-model-0.4.0-jar-with-dependencies.jar
```

Comment:
Is there a way to paramaterize the version?

Reply:
See comment just above, yes but there is no guarantee one version matches another.
It could be wildcarded for now?